src/PactNet/PactNet.csproj

```diff
@@ -49,6 +53,13 @@
       <CopyToOutputDirectory Condition="'$(IsOSX)'">PreserveNewest</CopyToOutputDirectory>
       <Visible>false</Visible>
     </Content>
+    <Content Include="$(MSBuildProjectDirectory)\..\..\build\osx\aarch64-apple-darwin\libpact_ffi.dylib">
+      <Link>libpact_ffi.dylib</Link>
+      <PackagePath>runtimes/osx-arm64/native</PackagePath>
+      <Pack>true</Pack>
+      <CopyToOutputDirectory Condition="'$(IsOSXArm64)'">PreserveNewest</CopyToOutputDirectory>
```

Comment:
I think it would be better to put the condition here, so that we just set IsOSX and IsArm64, then do the full condition in this section

Reply:
From latest changes all basic conditions are separate and then only for OSX it checks 2 conditions at once.